src/app/home/modules/bookmarks/bookmarks.component.ts

```diff
@@ -17,6 +17,8 @@ export class BookmarksComponent implements OnInit {
 		const isOutlineIcon = this.icon.substring(this.icon.length - 2, this.icon.length) == '-o'
 		if (this.icon.includes('fa-') && isOutlineIcon) { // If it's an outline icon from FA4 (fa-envelope-o), remove the 'fa' and 'o'
 			this.icon = this.icon.substring(3, this.icon.length - 2)
+		} else if (this.icon.includes('fa-')) {
+			this.icon = this.icon.substring(3);
 		}
```

Comment:
These could probably be combined into one statement:
```js
if (this.icon.includes('fa-') {
    // remove prefix
    if (isOutlineIcon) {
        // remove suffix
    }
}
```
Could maybe replace with a regex too, just another option https://regex101.com/r/FPXS47/1